Scripts/fastlane/Fastfile

```diff
@@ -190,6 +201,72 @@ import "./ScreenshotFastfile"
+  # build_and_upload_installable_build
+  # -----------------------------------------------------------------------------------
+  # This lane builds the app and upload it for adhoc testing 
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane build_and_upload_installable_build [version_long:<version_long>]
+  #
+  # Example:
+  # bundle exec fastlane build_and_upload_installable_build 
+  # bundle exec fastlane build_and_upload_installable_build build_version:123 
+  #####################################################################################
+  desc "Builds and uploads an installable build"
+  lane :build_and_upload_installable_build do | options |
+    alpha_code_signing
+
+    # Get the current build version, and update it if needed
+    version_config_path = "../../config/Version.internal.xcconfig"
+    versions = Xcodeproj::Config.new(File.new(version_config_path)).to_hash
+    build_version = versions["VERSION_LONG"]
+
+    if options.key?(:build_version)
+      build_version = options[:build_version]
+
+      UI.message("Updating build version to #{build_version}")
+
+      versions["VERSION_LONG"] = build_version
+      new_config = Xcodeproj::Config.new(versions)
+      new_config.save_as(Pathname.new(version_config_path))
+    end
+
+    gym(
+      scheme: "WordPress Alpha",
+      workspace: "../WordPress.xcworkspace",
+      export_method: "enterprise",
+      clean: true,
+      output_directory: "../build/",
+      derived_data_path: "../derived-data/alpha/",
+      export_team_id: ENV["INT_EXPORT_TEAM_ID"],
+      export_options: { method: "enterprise" })
+
+    sh("mv ../../build/WordPress.ipa \"../../build/WordPress Alpha.ipa\"")
+      
+    hockey(
+      api_token: get_required_env("HOCKEY_API_TOKEN"),        
+      public_identifier: get_required_env("HOCKEY_INSTALLABLE_BUILDS_PUBLIC_ID"),
+      notify: "0",
+      status: "2",
+      ipa: "../build/WordPress Alpha.ipa",
+      dsym: "../build/WordPress.app.dSYM.zip",
+    )
+
+    # Getting the download link from the HockeyApp Api, since `hockey(..)` only gives us the app link
+    # See documentation here: https://support.hockeyapp.net/kb/api/api-versions
+    url = "https://rink.hockeyapp.net/api/2/apps/#{get_required_env("HOCKEY_INSTALLABLE_BUILDS_PUBLIC_ID")}/app_versions"
+    result = open(url, {"X-HockeyAppToken" => get_required_env("HOCKEY_API_TOKEN")}).read
+    versions = JSON.parse(result)
+
+    download_url = versions["app_versions"].find {|v| v["version"] == build_version}["download_url"]
+    UI.message("Successfully built and uploaded installable build here: #{download_url}")
+
+    # Create a comment.json file so that Peril to comment with the build details, if this is running on CI
+    comment_body = "You can test the changes on this Pull Request by downloading it from HockeyApp [here](#{download_url}). If you need access to this, you can ask a maintainer to add you."
```

Comment:
This is the comment to be added to PRs with the download link. Unfortunately, we cannot open access up to everybody due to Apple's enterprise certificate restrictions. However, I think it is reasonable to give access to contributors who need it. With that in mind, do you agree this text makes sense?